### PR TITLE
RT123:-Added new proc value

### PR DESCRIPTION
**Pull request by moorthy818**
Added new proc value for new feature
image002

### CODE REVIEW

File: login_check.php [hunk 1]
```diff
@@ -2,5 +2,8 @@
 
 echo "Hello world";
 echo "Welcome To Juniper IAD Team";
+echo "Welcome To IAD Branch";
+echo "Welcome To MM Branch";
+echo "Thank you";
```
Comment (narayanan79) [inline login_check.php:7]: You might want to remove this line.

File: proc_test.txt [hunk 2]
```diff
@@ -0,0 +1,933 @@
...
+  IN  p_sequence         integer,
+  IN  p_job_name         varchar(256),
+  IN  p_task_name        varchar(256),
+  IN  p_job_id           integer,
+  IN  p_task_id          integer,
+  OUT p_return_msg       varchar(256),
+  OUT p_status           integer
+  )
+BEGIN
+
```
Comment (narayanan79) [inline proc_test.txt:12]: Good job on the comments

File: proc_test.txt [hunk 3]
```diff
@@ -0,0 +1,933 @@
...
+  DECLARE v_analyze_flag       integer;       -- analyze flag
+  DECLARE v_step               integer;       -- return code
+  DECLARE v_update_count       integer;       -- no of records updated
+  DECLARE v_insert_count       integer;       -- no of records inserted
+  DECLARE v_count              integer;       -- General counter
+  DECLARE v_sql_code           integer;       -- SQL Error Code for Audit Trail
+  DECLARE v_sql_error          varchar(255);  -- SQL Error Code for Audit Trail as varchar
+
+  --=====================================================
+  -- Variables
```
Comment (narayanan79) [inline proc_test.txt:40]: 💯

**Review by narayanan79 — APPROVED** [review]
LGTM (Looks Good To Me)

File: proc_test.txt (hunk 3 above)
Comment (moorthy818) [reply proc_test.txt:40]: Thanks a lot

**Comment by moorthy818** [comment]
Merging RT123

**Pull request merged**